dat/writers/create_reference_tables.py

```diff
@@ -0,0 +1,94 @@
+import pyspark
+from delta import *
+from pyspark.sql.types import *
+import os
+
+builder = (
+    pyspark.sql.SparkSession.builder.appName("MyApp")
+    .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension")
+    .config(
+        "spark.sql.catalog.spark_catalog",
+        "org.apache.spark.sql.delta.catalog.DeltaCatalog",
+    )
+)
+
+spark = configure_spark_with_delta_pip(builder).getOrCreate()
+
+def create_reference_table_1():
+    columns = ["letter", "number", "a_float"]
+    
+    data = [('a', 1, 1.1), ("b", 2, 2.2), ("c", 3, 3.3)]
+    rdd = spark.sparkContext.parallelize(data)
+    df = rdd.toDF(columns)
+
+    delta_table_path = "out/tables/generated/reference_table_1/delta"
+    expected_path = "out/tables/generated/reference_table_1/expected"
+    os.makedirs(expected_path)
+
+    df.repartition(1).write.format("delta").save(delta_table_path)
+    actual0 = spark.read.format("delta").load(delta_table_path)
+    actual0.toPandas().to_parquet(f"{expected_path}/table_content_v0.parquet")
+
+    data = [("d", 4, 4.4), ("e", 5, 5.5)]
+    rdd = spark.sparkContext.parallelize(data)
+    df = rdd.toDF(columns)    
+
+    df.repartition(1).write.format("delta").mode("append").save(delta_table_path)
+    actual1 = spark.read.format("delta").load(delta_table_path)
+    actual1.toPandas().to_parquet(f"{expected_path}/table_content_v1.parquet")
+
+
+def create_reference_table_2():
```

Comment:
It would be cool to have some human-readable description for each of these, so we have a general idea of what the case is testing.